pkg/tls/certRenewer.go

```diff
@@ -139,8 +139,7 @@ func (c *CertRenewer) WriteCACertToSecret(caPEM *PemPair, props CertificateProps
 					Name:      name,
 					Namespace: props.Namespace,
 					Annotations: map[string]string{
-						SelfSignedAnnotation: "true",
```

Comment:
This annotation is used in `addSecretFunc`, `updateSecretFunc` and `ReadTLSPair`. Will this break anything?